### PR TITLE
fix: Bumped up the dependency versions

### PR DESCRIPTION
**Pull request by OmAximani0**
- The versions are taken according PyPIs latest version
- Fixes: #100 
- The test are passing in my local environment
image

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
@OmAximani0 probably we should also bump the CI Python version to at least 3.8

**Comment by OmAximani0** [comment]
> @OmAximani0 probably we should also bump the CI Python version to at least 3.8

If that's ok, we should do that . Cause I was just updating the version according to `3.7`

**Comment by andrii-bodnar** [comment]
I guess this version is pretty old and other tools don't support it anymore, right?

**Comment by OmAximani0** [comment]
> I guess this version is pretty old and other tools don't support it anymore, right?

Yep, I also think so

**Comment by OmAximani0** [comment]
And @andrii-bodnar , I also think that we could support `3.10` and remove the `3.7`.

**Comment by andrii-bodnar** [comment]
@OmAximani0 yes, it would be great, thank you!

**Comment by andrii-bodnar** [comment]
@OmAximani0 please update the CI Python versions within the current PR

**Comment by andrii-bodnar** [comment]
I guess you can set the `3.8`, `3.9`, and `3.10` here https://github.com/crowdin/crowdin-api-client-python/blob/main/.github/workflows/python-package.yml#L18

**Comment by OmAximani0** [comment]
> I guess you can set the `3.8`, `3.9`, and `3.10` here https://github.com/crowdin/crowdin-api-client-python/blob/main/.github/workflows/python-package.yml#L18

Ok, got it :+1:

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.7, 3.8, 3.9 ]
+        python-version: [ 3.8, 3.9, 3.10 ]
```
Comment (andrii-bodnar) [inline .github/workflows/python-package.yml:18]:
```suggestion
        python-version: [ '3.8', '3.9', '3.10' ]
```

**Comment by OmAximani0** [comment]
I think we need to update `setup.cfg` like [this](https://stackoverflow.com/questions/74558565/flake8-error-code-supplied-to-ignore-option-does-not-match-a-z1-30)

**Comment by andrii-bodnar** [comment]
@OmAximani0 sounds pretty fair

**Review by andrii-bodnar — APPROVED** [review]
@OmAximani0 thank you!

**Pull request merged**